conf.yaml

```
@@ -1376,6 +1377,25 @@ contents:
               -
                 repo:   swiftype
                 path:   docs
+    -
+        title:      Enterprise Search
+        sections:
+          -
+            title:      Workplace Search
+            prefix:     en/enterprisesearch/workplacesearch
+            index:      workplace-search-docs/index.asciidoc
+            private:    1
+            current:    master
+            branches:   [ master ]
+            single:     1
+            tags:       Workplace Search/Reference
+            subject:    Workplace Search
+            sources:
+              -
+                repo:   enterprise-search
```

```suggestion
                repo:   enterprise-search-pubs
```

https://github.com/elastic/enterprise-search-pubs

---

It's based on the repo name set above. I changed it in both places. (Originally left the "pubs" off because it is longer than all the other names.)